service-front/assets/js/moj/moj.modules/moj.user-timeout.js

```diff
@@ -1,105 +0,0 @@
-// User Timeout module for LPA
-// Dependencies: moj, _, jQuery
-
-(function () {
-  'use strict';
-
-  moj.Modules.UserTimeout = {
-    timeoutDuration: 1000 * 60 * 55, // ms * s * m = 55 minutes
-    gracePeriod: 1000 * 60 * 5, // ms * s * m = 5 minutes
-    timeout: null,
-
-    init: function () {
-      _.bindAll(this, 'warning');
-
-      var startTime = this.time();
-      // check to see if user is logged in
-
-      if (false) {
```

Comment:
Note this condition never triggers, which suggests this is a redundant relic which was replaced with the session-timeout-dialog.js class.